src/platform/packages/shared/kbn-search-connectors/types/connectors.ts

```diff
@@ -227,6 +227,7 @@ export interface Connector {
   id: string;
   index_name: string | null;
   is_native: boolean;
+  deleted: boolean | false;
```

Comment:
This is type declaration, since it can be empty for old connectors it should be. Including `false` here doesn't make sense (as it's covered by boolean type), we don't set default values here
  
```suggestion
  deleted: boolean | null;
```

Reply:
Haha whoops. thought it's a default value